taskcluster/dotnet-package-opt-base.tyml

```diff
@@ -0,0 +1,91 @@
+$if: 'event.event in build.allowed'
+then:
+  taskId: ${taskcluster.taskId}
+  provisionerId: ${taskcluster.docker.provisionerId}
+  workerType: ${taskcluster.docker.workerTypeWin}
+  taskGroupId: ${taskcluster.taskGroupId}
+  schedulerId: ${taskcluster.schedulerId}
+  dependencies:
+    $map: { $eval: build.dependencies }
+    each(b):
+      $eval: as_slugid(b)
+  created: { $fromNow: '0 sec' }
+  deadline: { $fromNow: '1 day' }
+  expires:
+    $if: '(event.event == "push") || (event.event == "tag")'
+    then: { $fromNow: '6 months' }
+    else: { $fromNow: '7 days' }
+
+  extra:
+    nc_asset_name: { $eval: build.nc_asset_name }
+    github:
+      $if: '(event.event == "push") || (event.event == "tag")'
+      then: { $eval: taskcluster.github_events.merge }
+      else: { $eval: taskcluster.github_events.pull_request }
+
+  routes:
+    $if: '(event.event == "push") || (event.event == "tag")'
+    then:
+      { $eval: build.routes }
+
+  payload:
+    maxRunTime: { $eval: to_int(build.maxRunTime) }
+
+    env:
+      TC_MSYS_VERSION: 'MSYS_NT-6.3-9600'
+      MSYS: 'winsymlinks:nativestrict'
+      GIT_LFS_SKIP_SMUDGE: '1'
+
+    command:
+      - >-
+        "C:\Program Files\7-zip\7z.exe" x -txz -so msys2-base-x86_64.tar.xz |
+        "C:\Program Files\7-zip\7z.exe" x -o%USERPROFILE% -ttar -aoa -si
+      - .\msys64\usr\bin\bash.exe --login -cx "export THIS_BASH_PID=$$; ps -ef | grep '[?]' | awk '{print $2}' | grep -v $THIS_BASH_PID | xargs -r kill; exit 0"
+      - .\msys64\usr\bin\bash.exe --login -cx "pacman -Syu --noconfirm"
+      - echo .\msys64\usr\bin\bash.exe --login -cxe "
+        export LC_ALL=C &&
+        export PATH=\"/c/builds/tc-workdir/msys64/usr/bin:/c/Python36:/c/Program Files/Git/bin:/c/Program Files/7-Zip/:/c/Program Files (x86)/Windows Kits/10/bin/x64/:/c/Program Files/dotnet/:$PATH\" &&
+        export TASKCLUSTER_ARTIFACTS=\"$USERPROFILE/public\" &&
+        export TASKCLUSTER_TASK_DIR=\"/c/builds/tc-workdir/\" &&
+        (rm -fr $TASKCLUSTER_TASK_DIR/ ; mkdir $TASKCLUSTER_TASK_DIR) && cd $TASKCLUSTER_TASK_DIR &&
+        env &&
+        ln -s $USERPROFILE/msys64 $TASKCLUSTER_TASK_DIR/msys64 &&
+        (7z x -txz -so $USERPROFILE/home.tar.xz | 7z x -aoa -ttar -si ) &&
```

Comment:
looks like you are missing this one

Reply:
@stepkillah you need to either add the missing mount entry or remove the extraction of home.tar.xz: this contains only tensorflow prebuilt dependenciees, so if your build script ONLY build nuget/.net parts, it's likely you just dont need it